srcpkgs/zutty/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'zutty'
+pkgname=zutty
+version=0.9
+revision=1
+build_style=waf3
+hostmakedepends="pkg-config"
+makedepends="python3-devel libglvnd-devel MesaLib-devel freetype-devel libXmu-devel"
+depends="libglvnd freetype libXmu font-misc-misc"
```

Comment:
The first 3 should be detected automatically.

Reply:
What do you mean by the first 3?
It's my first contribution and don't know the ins and outs of package building.

Reply:
The first three packages listed in `depends+`.